configure

```diff
@@ -532,7 +542,7 @@ case "$CXX_TMP:$SYS" in
   ;;
   *clang*:"Droid")
     LINK_START_GROUP="-Wl,--start-group"
-    LINK_END_GROUP="-Wl,--end-group"
+    LINK_END_GROUP="--end-group"
```

Comment:
Pretty sure that still wants `-Wl,`.

Reply:
Most likely due to editing the link end group to address some link order errors and having a mistype.